pkg/kubectl/cmd/create.go

```diff
@@ -89,18 +94,28 @@ func RunCreate(f *cmdutil.Factory, cmd *cobra.Command, out io.Writer, options *C
 	}
 
 	mapper, typer := f.Object()
-	r := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
+	b := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
 		Schema(schema).
 		ContinueOnError().
 		NamespaceParam(cmdNamespace).DefaultNamespace().
 		FilenameParam(enforceNamespace, options.Filenames...).
-		Flatten().
-		Do()
+		Flatten()
+	transformArg := cmdutil.GetFlagString(cmd, "transforms")
+	if len(transformArg) > 0 {
```

Comment:
I don't think we want to do that.  I think that the set of transforms is going to be parameterized by the factory (as it is with the `GeneratorTransform` here, and the factory isn't available inside of the `Builder` object.  I suppose I could add it...